packages/volto/cypress/tests/core/controlpanels/upgrade.js

```diff
@@ -39,7 +39,7 @@ describe('Upgrade Site Tests', () => {
       body: {
         ...getUpgradeNeedsUpgrade,
       },
-    }).as('getSystemNeedsUpdate');
+    }).as('getUpgradeNeedsUpgrade');
```

Comment:
Renamed, but then not used? watch out

Reply:
@sneridagh You have a point here. I changed it because the way I understand this file, there's a name clash. The original version of the test is:

```javascript
  it('As manager, I can click upgrade site, the upgrade control panel shows', function () {
    cy.intercept('GET', '/**/@system', {
      statusCode: 200,
      body: {
        ...getsystemNeedsUpgrade,
      },
    }).as('getSystemNeedsUpdate');
    cy.intercept('GET', '/**/@upgrade', {
      statusCode: 200,
      body: {
        ...getUpgradeNeedsUpgrade,
      },
    }).as('getSystemNeedsUpdate');
    cy.navigate('controlpanel');
    cy.wait('@getSystemNeedsUpdate');

    cy.findByText('Please continue with the upgrade.').click();
    cy.get('.content-area').contains(
      'The site configuration is outdated and needs to be upgraded.',
    );
  });
```

If I understand correctly there are two different calls intercepted with the alias `getSystemNeedsUpdate` and I doesn't feel right. On the other hand, if I change it where do I wait for the second one? Maybe after the click?

Reply:
I tried adding an additional `wait` after the click, I am not sure it is needed at all, but the request is actually made to the backend so I think it is worth waiting for it to actually happen, at least in the current state of things

Reply:
I see, maybe we need to revisit this at some point.